File: 01. SourceCode/ChildFund/ChildProfile.Model/Model/ObjectInputModel/ObjectBaseModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ChildProfiles.Model
{
    public class ObjectBaseModel
    {
        public List<ObjectInputModel> ListObject { get; set; }
        /// <summary>
        /// Tên nội dung khác
        /// </summary>
        public string OtherName { get; set; }
        /// <summary>
        /// Giá trị khác
        /// </summary>
        public string OtherValue { get; set; }

        /// <summary>
        /// Tên nội dung khác nữa
        /// </summary>
        public string OtherName2 { get; set; }
        /// <summary>
        /// Giá khác nữa
        /// </summary>
        public string OtherValue2 { get; set; }
    }
}

File: 01. SourceCode/IOS/childprofile/childprofile/Models/CreateChildProfile/ObjectBaseModel.swift
//
//  ObjectBaseModel.swift
//  childprofile
//
//  Created by Thanh Luu on 1/20/19.
//  Copyright © 2019 childfund. All rights reserved.
//

import Foundation

class ObjectBaseModel: Codable {
    var listObject: [ObjectInputModel]
    var otherName: String
    var otherValue: String
    var otherName2: String
    var otherValue2: String
    
    enum CodingKeys: String, CodingKey {
        case listObject = "ListObject"
        case otherName = "OtherName"
        case otherValue = "OtherValue"
        case otherName2 = "OtherName2"
        case otherValue2 = "OtherValue2"
    }
    
    required init(from decoder: Decoder) throws {
        let container = try decoder.container(keyedBy: CodingKeys.self)
        self.listObject = try container.decodeIfPresent([ObjectInputModel].self, forKey: .listObject) ?? []
        self.otherName = try container.decodeIfPresent(String.self, forKey: .otherName) ?? ""
        self.otherValue = try container.decodeIfPresent(String.self, forKey: .otherValue) ?? ""
        self.otherName2 = try container.decodeIfPresent(String.self, forKey: .otherName2) ?? ""
        self.otherValue2 = try container.decodeIfPresent(String.self, forKey: .otherValue2) ?? ""
    }
}
